utils/query-graph/query-graph.js

```diff
@@ -16,14 +18,15 @@ class QueryGraph {
    *
    * @param {Array<Package>} packages An array of Packages to build the graph out of
    * @param {String} [opts.graphType="allDependencies"] "dependencies" excludes devDependencies from graph
+   * @param {String} [opts.localDependencies="auto"] Valid values are "force" or "explicit"
    * @param {Boolean} [opts.rejectCycles] Whether or not to reject cycles
    * @constructor
    */
   constructor(packages, opts) {
     const options = QueryGraphConfig(opts);
 
     // Create dependency graph
-    this.graph = new PackageGraph(packages, options.graphType);
+    this.graph = new PackageGraph(packages, options.graphType, options.localDependencies);
```

Comment:
[run-topologically](https://github.com/lerna/lerna/blob/master/utils/run-topologically/run-topologically.js#L33) also needs this option passed through, as unfortunately most of its callsites are passing custom config, not the whole command-resolved `options` object...